VAMobile/src/store/api/demo/mocks/secureMessaging.json

```diff
@@ -424,7 +424,7 @@
         "messageId": 2092809,
         "category": "MEDICATIONS",
         "subject": "Naproxen side effects",
-        "body": "Upset stomach is a common side effect of this medication.  Mild stomach pain is normal, but if you are having severe stomach pains, please let us know or seek in-person care. If you need further assistance that isn't an emergency please call 8006982411 or 800-698-2411 or (800)698-2411, or (800)-698-2411 or 800 698 2411 or +8006982411 or +18006982411 or 1-800-698-2411. For more information on the side effects please go to one of these links: https://www.va.gov/ or https://rb.gy/riwea or https://va.gov or http://www.va.gov/ or https://www.va.gov/education/about-gi-bill-benefits/ or www.va.gov or www.google.com or google.com  you can also email me at test@va.gov or mailto:test@va.gov or you can come to the office at iOS: http://maps.apple.com/?q=Mexican+Restaurant&sll=50.894967,4.341626&z=10&t=s or android: http://maps.google.com/?q=50.894967,4.341626",
+        "body": "Upset stomach is a common side effect of this medication.\nMild stomach pain is normal, but if            you are having severe stomach pains, please let us know or seek in-person care.\tIf you need further assistance that isn't an emergency please call 8006982411 or 800-698-2411 or (800)698-2411, or (800)-698-2411 or 800 698 2411 or +8006982411 or +18006982411 or 1-800-698-2411. For more information on the side effects please go to one of these links: https://www.va.gov/ or https://rb.gy/riwea or https://va.gov or http://www.va.gov/ or https://www.va.gov/education/about-gi-bill-benefits/ or www.va.gov or www.google.com or google.com you can also email me at test@va.gov or mailto:test@va.gov or you can come to the office at iOS: http://maps.apple.com/?q=Mexican+Restaurant&sll=50.894967,4.341626&z=10&t=s or android: http://maps.google.com/?q=50.894967,4.341626",
```

Comment:
Thanks for updating demo mode to show these changes. Could you add/update the unit tests as well?

Reply:
unit tests don't typically play nice with spacing. I feel this one is likely fine to manually verify for testing purposes.